code/source/playfab/PlayFabPluginManager.cpp

```diff
@@ -84,7 +84,7 @@ namespace PlayFab
 #ifdef PLAYFAB_PLATFORM_XBOX
         return std::make_shared<PlayFabIXHR2HttpPlugin>();
 #elif defined(PLAYFAB_PLATFORM_WINDOWS)
-        return std::make_shared<PlayFabWinHttpPlugin>();
+        return std::make_shared<PlayFabHCHttpPlugin>();
 #elif defined(PLAYFAB_PLATFORM_IOS)
```

Comment:
the plan is to move away from IPlayFabPlugin and just use LibHttpClient right for all the other platforms?

Reply:
Yes, this is the plan.  In fact, the whole http plugin concept will most likely get removed here (libHttpClient already has a similar idea, so titles really wanted custom http stack they could do it at that level)

---
In reply to: [590805142](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r590805142) [](ancestors = 590805142)